web/settings/instance.go

```diff
@@ -43,6 +43,7 @@ func getInstance(c echo.Context) error {
 		return err
 	}
 	doc.M["locale"] = instance.Locale
+	doc.M["onboarding_finished"] = instance.OnboardingFinished
```

Comment:
I was hesitant to allow mutation from this endpoint and decided to set the flag on the `/context/onboarded` handler which provide the redirect at the end of the onboarding: see https://github.com/cozy/cozy-stack/pull/894/files#diff-0ff4658f63295d93dc43ccf67d16d07cR43